### ISSUE TITLE
(Feature request) Mark read on scroll 

### ISSUE THREAD

**Issue by dtantono1** [sheodox/alexandrite#74]
Hi
on Memmy for Lemmy or voyager for Lemmy. there is a feature called mark read on scroll that ensure to keep our home screen updated and I think it will be great that Alexandrite have this feature. What do you think?

**Comment by sheodox** [comment]
Yeah I'm planning on making a setting for that at some point, I've been thinking about how I'll do it lately.

**Comment by pitrisss** [comment]
+1, this would be a really great addition.
